.travis.yml

```diff
@@ -1,89 +1,96 @@
 addons:
```

Comment:
Planning to merge this one.

@moretea please add a review to .travis.yml. The docs aren't ready but I want to have the doc structure in develop asap.